app/services/application_state_change.rb

```diff
@@ -70,4 +70,8 @@ def persist_workflow_state(new_state)
   def self.valid_states
     workflow_spec.states.keys
   end
+
+  def references_complete?
+    application_choice.application_form.references_complete?
+  end
```

Comment:
Is this method being used anywhere anymore?

Reply:
good catch, I've removed it